src/main/java/org/gradle/profiler/studio/LauncherConfigurationParser.java

```diff
@@ -44,24 +42,18 @@ public LaunchConfiguration calculate(Path studioInstallDir, int studioPluginPort
         Path supportJar = GradleInstrumentation.unpackPlugin("instrumentation-support").toPath();
         Path protocolJar = GradleInstrumentation.unpackPlugin("client-protocol").toPath();
         Path studioPlugin = GradleInstrumentation.unpackPlugin("studio-plugin").toPath();
-        Path studioPluginsDir = newPluginTempDir();
 
-        String studioPluginsDirPath = studioPluginsDir.toAbsolutePath().toString();
+        Path studioPluginsDir = studioSandbox.getPluginsDir();
+        Path studioLogsDir = studioSandbox.getLogsDir();
+        studioSandbox.getConfigDir().ifPresent(path -> systemProperties.put("idea.config.path", path.toString()));
+        studioSandbox.getSystemDir().ifPresent(path -> systemProperties.put("idea.system.path", path.toString()));
```

Comment:
Readability: lines 46-47 should be swapped with lines 48-49, no?

Reply:
Done